src/betterproto/plugin/__init__.py

```diff
@@ -0,0 +1 @@
+from .main import main
```

Comment:
```suggestion
from betterproto.plugin.main import main
```

Or better yet, drop this in favour of consolidating `main.py` into `__main__.py` and update `pyproject.toml` to use `betterproto.plugin.__main__:main`.